cmd/tests/reprolang/bindings/go/repro/scip.go

```diff
@@ -17,6 +19,11 @@ func (i *identifier) occurrence(roles scip.SymbolRole) *scip.Occurrence {
 func (s *reproSourceFile) symbols() []*scip.SymbolInformation {
 	var result []*scip.SymbolInformation
 	for _, def := range s.definitions {
+		fmt.Println("def.name.value", def.name.value)
+		if strings.Index(def.name.value, "NoSymbolInformation") >= 0 {
+			fmt.Println("BOOM")
+			continue
+		}
```

Comment:
```suggestion
```

It seems like these lines are for debugging?

Reply:
Good catch! I opened the PR in a rush. Fixed now